package.json

```
@@ -48,7 +48,7 @@
         ]
     },
     "dependencies": {
-        "@flyteorg/flyteidl": "1.1.16"
+        "@flyteorg/flyteidl": "https://github.com/flyteorg/flyteidl.git#feature/gate-nodes"
```

We don't need this in prod, please either check out those files or push a revert of the version
```suggestion
        "@flyteorg/flyteidl": "1.1.16"
```